types/src/on_chain_config/aptos_features.rs

```diff
@@ -158,4 +161,8 @@ impl Features {
     pub fn is_open_id_signature_enabled(&self) -> bool {
         self.is_enabled(FeatureFlag::OPEN_ID_SIGNATURE)
     }
+
+    pub fn is_refundable_bytes_enabled(&self) -> bool {
```

Comment:
I think we should document our features better: you have a great summary in this PR but having something written in the code would help?